tutorial/tutorial-gabc-01.html

```diff
@@ -248,7 +248,7 @@
 </pre>
 </div>
 
-<p>Here's the result after processing by gregorio and LaTeX:</p>
+<p>Here's the result after processing by gregorio and <span class="latex">L<span class="alpha">a</span>T<span class="epsilon">e</span>X</span>:</p>
```

Comment:
Also, there is a reference to step numbers, but the steps are not actually numbered.